entity-view/impl/src/main/java/com/blazebit/persistence/view/impl/mapper/ViewMapper.java

```diff
@@ -409,9 +420,6 @@ public Key(ManagedViewTypeImplementor<S> sourceType, ManagedViewTypeImplementor<
 
         public static <X, Y> Key<X, Y> create(ViewMetamodel metamodel, Class<X> sourceEntityViewClass, Class<Y> targetEntityViewClass, String targetConstructorName, boolean ignoreMissingAttributes, boolean markNew) {
             ManagedViewTypeImplementor<X> sourceViewType = (ManagedViewTypeImplementor<X>) metamodel.managedView(sourceEntityViewClass);
-            if (sourceViewType == null) {
-                throw new IllegalArgumentException("Unknown source view type: " + sourceEntityViewClass.getName());
-            }
```

Comment:
Please revert this change. I'd prefer to handle this in the create method that receives the object.